doc/interop-test-descriptions.md

```diff
@@ -1057,6 +1067,12 @@ This test must be configurable via a few different command line flags:
 * `soak_min_time_ms_between_rpcs`: The minimum time in milliseconds between
   consecutive RPCs. Useful for limiting QPS.
 
+* `soak_num_threads`: Specifies the number of threads to use for concurrently 
+  executing the soak test. Each thread performs a portion of the total 
```

Comment:
Instead of "a portion of the total soak_iterations", let's say "Each thread performs `soak_iterations / soak_num_threads` RPCs"